binding/multipart_form_mapping.go

```diff
@@ -39,6 +41,27 @@ func setByMultipartFormFile(value reflect.Value, field reflect.StructField, file
 			return true, nil
 		}
 	case reflect.Slice:
+		switch value.Interface().(type) {
+		case []byte:
+			fd, err := files[0].Open()
+			if err != nil {
+				return false, err
+			}
+			defer fd.Close()
+			c, err := ioutil.ReadAll(fd)
+
+			if ConstructionFailure {
```

Comment:
In my opinion, it is a bad practice when the code depends on the implementation of the tests.

Reply:
ok，I will find a way. @vkd  There are other ways to bypass the cover detection ?

Reply:
@vkd 

https://github.com/golang/go/blob/master/src/context/context.go#L243

Add some test code to the official code, someone has done this, such as rsc, in the standard library. So, I don't think this piece needs to be modified.

Reply:
In your example, there is the different case, because the behavior of a method does not change.